studio/components/layouts/ProjectLayout/ProjectPausedState.tsx

```diff
@@ -29,6 +30,8 @@ const ProjectPausedState = ({ product }: ProjectPausedStateProps) => {
   const { data: subscription } = useProjectSubscriptionV2Query({ projectRef: ref })
   const isFreePlan = subscription?.plan?.id === 'free'
   const billedViaOrg = Boolean(selectedOrganization?.subscription_id)
+  const { data: allOverdueInvoices } = useOverdueInvoicesQuery({ enabled: IS_PLATFORM })
+  const haveOverdueInvoices: boolean = (allOverdueInvoices?.length ?? 0) > 1
```

Comment:
The overdue invoices endpoint doesn't work here, as it covers ALL projects a user is a member of. It's used on the front page where all orgs are displayed.

For this use-case, we need to only check for the currently selected org

Reply:
We could also remove the check from the frontend entirely and move it to the backend, then just react to the appropriate error and display a warning. Then we can control this logic on the backend, without duplicating it here

Reply:
Got it on just implementing another endpoint in the backend.
For users without org-level billing, I need to check on a project level for now. Do I check for the `stripe_customer_id` related to the org only? How do I find on a project level?

Reply:
All projects in an org point to a single Stripe customer, so it's okay to not let a user unpause ANY project if the related org has open invoices (check works for both org and non org level billing).

There is not necessarily a need to implement a new endpoint at all. See 

> We could also remove the check from the frontend entirely and move it to the backend, then just react to the appropriate error and display a warning. Then we can control this logic on the backend, without duplicating it here

This would not require any new endpoint. When the user presses unpause, it would show an error about outstanding invoices and could then display the CTA. It's a bit delayed then, but reduces code complexity and avoids spreading the logic in multiple places